episodes/choosing-acquisition-settings.md

```diff
@@ -0,0 +1,546 @@
+---
+title: 'Choosing acquisition settings'
+teaching: 30
+exercises: 10
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What are the key factors to consider when choosing acquisition settings?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Explain some examples of acquisition settings, and factors to consider when 
+choosing them
+
+- Explain the difference between resolution and pixel size
+
+- Describe some quality control steps e.g. monitoring the image histogram
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+In the last episode we looked at some of the main steps to designing a light 
+microscopy experiment:
+
+1. Define your research question
+
+2. Define what you need to observe to answer that question
+
+3. Define what you need to measure to answer that question
+
+4. Choose a light microscopy method that fits your data needs
+
+5. Choose acquisition settings that fit your data needs
+
+In this episode, we'll focus on the last step - choosing acquisition settings - 
+as well as looking at some early quality control steps.
+
+## Acquisition settings
+
+Once you've chosen a light microscope to use, there are a wide variety of 
+acquisition settings that can be adjusted. For example, magnification, laser 
+power, exposure time... Different combinations of settings will be best for 
+different samples and research questions. For example, one combination of 
+settings may be best for rapid live cell imaging, while another may be best for 
+high resolution imaging of fixed samples.
+
+There are many different factors that are affected by acquisition settings, but 
+here are some of the main ones to consider:
+
+## Spatial resolution
+
+Spatial resolution is the resolution in 2D (x and y), or 3D (x, y and z). It is 
+defined as the smallest distance between two points on a sample that can still 
+be seen as separate entities. For example, a resolution of one micrometre would 
```

Comment:
Thanks @davecash75 - I've changed it to: 'Spatial resolution is defined as the smallest distance between two adjacent 
points on a sample that can still be seen as separate entities. In 2D, this is measured in x and y, while in 3D it is measured in x, y and z.'. I wanted to avoid mentioning voxels (+ pixels) here, as they can be much smaller than the real spatial resolution.

Reply:
Of course, makes sense.